task60_PageObject/src/test/java/SingletonClass/WebDriverSingleton.java

```diff
@@ -0,0 +1,44 @@
+package SingletonClass;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.concurrent.TimeUnit;
+
+public class WebDriverSingleton {
```

Comment:
Okay, let it be like this for now, but it is not singleton. We will cover this out of scope of this task